source/assets/stylesheets/styles/_bitesize.scss

```diff
@@ -82,14 +115,90 @@
   padding-bottom: 16px;
 
   &:last-of-type {
+    padding-bottom: 16px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    font-size: 1.96vw;
+  }
+
+  @include breakpoint($max-tablet-container) {
+    font-size: 1.18em;
+  }
+}
+
+.bitesize__list-item--white {
+  @extend .bitesize__list-item;
+  color: $bitesize-white-text-color;
+
+  p {
     padding-bottom: 8px;
   }
 
+  span {
+    font-size: 0.7em;
+    font-weight: 300;
+    letter-spacing: 0.07em;
+  }
+}
+
+.bitesize__text--large {
+  color: $bitesize-white-text-color;
+  font-size: 2em;
+  padding-bottom: 10px;
+  padding-top: 20px;
+  text-align: center;
+
+  i {
```

Comment:
On first glance I thought this was styling for the `italic` tag, but realised it was actually for FontAwesome. Perhaps we can add a new class to avoid confusion and better describe what it is here we are styling.